src/lib/server.ts

```diff
@@ -67,6 +95,7 @@ export const HttpRequest = Record({
     body: blob,
     certificate_version: Opt(nat16)
 });
+export type HttpRequest = typeof HttpRequest.tsType;
```

Comment:
Why can't we use `.tsType` everywhere else?

Reply:
I believe it's because the other ones have the generic Token